source/jsonnet/northern-ireland/individual/blocks/identity-and-health/past_usual_address_outside_uk.jsonnet

```diff
@@ -0,0 +1,46 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local question(title) = {
+  id: 'past-usual-address-outside-uk-question',
+  title: title,
+  type: 'General',
+  answers: [
+    {
+      id: 'past-usual-address-outside-uk-answer',
+      label: 'Current name of country',
+      description: 'Enter your own answer or select from suggestions',
+      max_length: 100,
+      mandatory: false,
+      suggestions_url: 'https://cdn.eq.census-gcp.onsdigital.uk/data/v1.0.0/countries-of-birth.json',
+      type: 'TextField',
+    },
+  ],
+};
+
+{
+  type: 'Question',
+  id: 'past-usual-address-outside-uk',
+  question_variants: [
+    {
+      question: question('In which country outside the UK was your usual address one year ago?'),
```

Comment:
<img width="1239" alt="Screenshot 2020-05-19 at 11 28 07" src="https://user-images.githubusercontent.com/53475968/82316084-3597c500-99c4-11ea-8f51-8ea6d9e7bdcf.png">

Reply:
Whats the change? I can't tell from this comment. Thank you

Reply:
It is a small typo: `outside the UK` instead of `outside of the UK`.